### PR TITLE
Marketplace

### PR DESCRIPTION
**Pull request by 0xAshish**

### CODE REVIEW

**Comment by 0xAshish** [comment]
closing it, since it's going with refactoring branch https://github.com/maticnetwork/contracts/pull/67.

File: contracts/child/ChildERC721.sol [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
-import { ERC721Full } from "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";
-
+import "./ERC721.sol";
 import "./ChildToken.sol";
 import "./IParentToken.sol";
 
 
-contract ChildERC721 is ChildToken, ERC721Full {
+contract ChildERC721 is ChildToken, ERC721 {
```
Comment (jdkanani) [inline contracts/child/ChildERC721.sol]: Use `ERC721Full` and remove explicit `_transferFrom`.

Comment (jdkanani) [reply contracts/child/ChildERC721.sol]: cc @0xAshish @atvanguard

Comment (0xAshish) [reply contracts/child/ChildERC721.sol]: this one is old PR, new one which got merged has ERC721Full.

**Pull request closed without merging**